src/make_ntuples.c

```diff
@@ -81,6 +83,17 @@ int run(char * in_filename, bool use_fmt, bool debug, int nevn, int run_no, doub
         }
 
         // Process DIS event.
+        // Checking existence of trigger electron in event
+        particle p_el = particle_init();
+        for (UInt_t pos = 0; pos < rtrk.index->size(); ++pos) {
+            p_el = use_fmt ? particle_init(&rpart, &rtrk, &ftrk, pos)
+                           : particle_init(&rpart, &rtrk, pos);
+            if(p_el.is_trigger_electron) break;
+        }
+        // Guarding statement
+        if(!p_el.is_trigger_electron) continue;
```

Comment:
I think we already discussed this, but it'd be better if you didn't drop a whole event if you can't find a trigger electron.

Reply:
Lets fight to the death to solve this one. Shall we?